### PR TITLE
djangocon-2009

### PR DESCRIPTION
**Pull request by logston**

### CODE REVIEW

File: data/djangocon-2009/videos/djangocon-2009--django-is-obsolete--but-so-is-eve.json [hunk 1]
```diff
@@ -1,29 +1,29 @@
...
-  "language": "English", 
-  "copyright_text": "Creative Commons Attribution-NonCommercial-ShareAlike 3.0", 
-  "thumbnail_url": "http://a.images.blip.tv/Robertlofthouse-DjangoIsObsoleteButSoIsEverythingElse277.png", 
-  "duration": null, 
+  "id": 33,
+  "category": "DjangoCon 2009",
+  "slug": "djangocon-2009--django-is-obsolete--but-so-is-eve",
+  "title": "Django is obsolete (but so is everything else)",
+  "summary": "",
+  "description": "Django is obsolete (but so is everything else)\n\nPresented by Avi Bryant\n\n\\*\\* Keynote for DjangoCon 2009.\\*\\*\n",
```
Comment (redapple) [inline data/djangocon-2009/videos/djangocon-2009--django-is-obsolete--but-so-is-eve.json:7]: `\*\* Keynote for DjangoCon 2009.\*\*` (originally as `** Keynote for DjangoCon 2009.**`) should probably be `**Keynote for DjangoCon 2009.**` in reST (strong emphasis)

File: data/djangocon-2009/videos/djangocon-2009--introduction-to-geodjango.json [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
-  "language": "English", 
-  "copyright_text": "Creative Commons Attribution-NonCommercial-ShareAlike 3.0", 
-  "thumbnail_url": "http://a.images.blip.tv/Robertlofthouse-IntroductionToGeoDjango145.png", 
-  "duration": 15, 
+  "id": 18,
+  "category": "DjangoCon 2009",
+  "slug": "djangocon-2009--introduction-to-geodjango",
+  "title": "Introduction to GeoDjango",
+  "summary": "Gives an introduction to GeoDjango.\n",
+  "description": "\r\n",
```
Comment (redapple) [inline data/djangocon-2009/videos/djangocon-2009--introduction-to-geodjango.json:7]: empty-line "description" could be removed/set to ""

File: data/djangocon-2009/videos/djangocon-2009--real-time-web-and-other-buzzwords.json [hunk 3]
```diff
@@ -1,28 +1,28 @@
...
   "tags": [
-    "comet", 
-    "djangocon", 
+    "comet",
+    "djangocon",
     "djangocon2009"
-  ], 
-  "speakers": [], 
+  ],
+  "speakers": [],
```
Comment (redapple) [inline data/djangocon-2009/videos/djangocon-2009--real-time-web-and-other-buzzwords.json:26]: For the record, the speaker was Chris Wanstrath

File: data/djangocon-2009/videos/djangocon-2009--restful-ponies.json [hunk 4]
```diff
@@ -1,28 +1,28 @@
...
   "tags": [
-    "djangocon", 
-    "djangocon2009", 
+    "djangocon",
+    "djangocon2009",
     "rest"
-  ], 
-  "speakers": [], 
+  ],
+  "speakers": [],
```
Comment (redapple) [inline data/djangocon-2009/videos/djangocon-2009--restful-ponies.json:26]: was Mike Malone (@mjmalone) (source: http://controlroom.blogspot.fr/2009/09/more-from-djangocon.html)

**Comment by redapple** [comment]
Missing speakers (also missing in original Markdown) can be found on https://web.archive.org/web/20091013114444/http://www.djangocon.org/2009/conference/schedule/ and http://lanyrd.com/2009/djangocon-us/

To fix in another PR I believe

**Comment by codersquid** [comment]
Thanks @redapple I've created 110 to track the fixes.

**Pull request merged**